src/extractor/extractor.cpp

```diff
@@ -93,7 +94,8 @@ int Extractor::run(ScriptingEnvironment &scripting_environment)
         util::SimpleLogger().Write() << "Threads: " << number_of_threads;
 
         ExtractionContainers extraction_containers;
-        auto extractor_callbacks = util::make_unique<ExtractorCallbacks>(extraction_containers);
+        auto extractor_callbacks =
+            util::make_unique<ExtractorCallbacks>(extraction_containers, turn_lane_map);
```

Comment:
Hmm looks like we're calling .reset below to delete the callbacks.. it's keeping the deduplication map around, only refs otherwise. Kind of ugly..